django/contrib/admindocs/templates/admin_doc/model_detail.html

```diff
@@ -55,14 +55,14 @@ <h3>{% translate 'Methods with arguments' %}</h3>
 <table class="model">
 <thead>
 <tr>
-    <th>{% translate 'Method' %}</th>
-    <th>{% translate 'Arguments' %}</th>
-    <th>{% translate 'Description' %}</th>
+    <th scope="col">{% translate 'Method' %}</th>
+    <th scope="col">{% translate 'Arguments' %}</th>
+    <th scope="col">{% translate 'Description' %}</th>
 </tr>
 </thead>
 <tbody>
 {% for method in methods|dictsort:"name" %}
-<tr>
+<tr scope="row">
     <td>{{ method.name }}</td>
```

Comment:
I think we should add `scope="row"` to `{{ field.name }}` and `{{method.name}}`.

Reply:
I am not sure here. If we do, we need to make it `th` because `scope` is deprecated on `td`. It should also be semantically and visually a heading.

Reply:
Ah right, we don't have `<th>` here. Thanks :+1: